cmd/lassie/version.go

```diff
@@ -23,13 +18,14 @@ var versionCmd = &cli.Command{
 	Action: versionCommand,
 }
 
+var defaultVersion string = "(devel)"
```

Comment:
perhaps the default should still be semver

```suggestion
var defaultVersion string = "v0.0.0-dev"
```

Reply:
The suffix `-dirty` is what i see used across other projects like Boost. It'd be good to get short commit ID in there if we don't already.

Reply:
So this is setup in a way where the `defaultVersion` will only be printed from the `lassie version` command when using a Lassie binary that was not built setting the `version.Version` variable via ldflags. So locally, manually built binaries via `go build` will output this. `(devel)` was chosen as it's the default used for golang build info

Getting a git commit in there might be a bit difficult. However, using `goreleaser release --snapshot` locally will properly build with a version snapshot and short commit.